### ISSUE TITLE
Cross-check of formalisms

### ISSUE THREAD

**Issue by mmikhasenko** [KaiHabermann/decayangle#63]
Here we agree on setup to proceed with matching of all formalisms

## Kinematics

Let's consider a reaction with quantum numbers, labeled as 0 -> 1,2,3

- masses: `6.32397 -> 1.0, 2.0, 3.0`
- spins: `1/2 -> 1/2, 1, 0`
- chains:
   - chain `(12)3`: j=1/2, with `ls=(1,1/2)`, and `LS=(1, 3/2)`
   - chain `(23)1`: j=2, with `ls=(2,1)`, and `LS=(1, 3/2)`

## The test point

- orientation angles `(phi_rf, cos_theta_rf, psi_rf) = (0.3, 0.4, 0.5)`. The value polar angle is determined through the `cos` function. 
- Dalitz point is specified as `m23sq`, `m31sq`, `m12sq`, respectively,
```julia
σs = ((26.57159046, 17.86811729, 9.55283383))
```
- spins projections, let's say `lam0=-1/2` `lam1= 1/2`, `lam2=0`, and `lam3=0`

## Amplitude
computed as `A_{c=3} + A_{c=1} * W_{1(3)}`, with the first term in the sum being called `term1`, and the second term with a summation over wigner rotation indices is being referred to as the `term2`.

Values yielded by the formalisms printed for two terms separately:
 - DPD (gist https://gist.github.com/mmikhasenko/1d1f59664d1aefc3874cddab8670c736):
```
term1 = -0.20903167421394864 - 0.9790486016833232im
term2 = 3.704279900338359 - 2.486666937280139im
```
 - helicity
```
term1: (xxx, xxx)
term2: (xxx, xxx)
```
- minus_phi
```
term1: (xxx, xxx)
term2: (xxx, xxx)
```
- canonical
```
term1: (xxx, xxx)
term2: (xxx, xxx)
```

**Comment by mmikhasenko** [comment]
Just computed for the aligned kinematics,

<img width="408" alt="image" src="https://github.com/user-attachments/assets/ff6969fa-2f41-4b86-9d2d-d3632754f81a">

**Comment by mmikhasenko** [comment]
With the overall rotation,

<img width="749" alt="image" src="https://github.com/user-attachments/assets/a53a14f2-6db9-4ecc-b24e-af7bbb2e6912">

**Comment by mmikhasenko** [comment]
propagated the DPD values to the header together with a notebook reference, https://gist.github.com/mmikhasenko/1d1f59664d1aefc3874cddab8670c736)

**Comment by KaiHabermann** [comment]
The LS couplings you gave appear to be inconsistent

> Just computed for the aligned kinematics,
> <img alt="image" width="408" src="https://private-user-images.githubusercontent.com/22725744/361410133-ff6969fa-2f41-4b86-9d2d-d3632754f81a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ2NzgyODQsIm5iZiI6MTcyNDY3Nzk4NCwicGF0aCI6Ii8yMjcyNTc0NC8zNjE0MTAxMzMtZmY2OTY5ZmEtMmY0MS00Yjg2LTlkMmQtZDM2MzI3NTRmODFhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODI2VDEzMTMwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWMzNWRmNGE5OTliYWY5N2YxN2NlOWU4YjQwZWJlMWIwNGFhZGNiNTMwZjZhYWI5NWU3ZWY4MWY3ZWQ4NWJlMGMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.7XKAbyJ6JLZYyqw8cm6JsDE7ejsEOl9QTmzpEI4lohA">

I dont really know, what I am looking at there. What are the matrix elements?

**Comment by KaiHabermann** [comment]
> Here we agree on setup to proceed with matching of all formalisms
> ## Kinematics
> 
> Let's consider a reaction with quantum numbers, labeled as 0 -> 1,2,3
> 
>     * masses: `6.32397 -> 1.0, 2.0, 3.0`
> 
>     * spins: `1/2 -> 1/2, 1, 0`
> 
>     * chains:
>       
>       * chain `(12)3`: j=1/2, with `ls=(1,1/2)`, and `LS=(1, 3/2)`
>       * chain `(23)1`: j=2, with `ls=(2,1)`, and `LS=(1, 3/2)`
> 
> 
> ## The test point
> 
>     * orientation angles `(phi_rf, cos_theta_rf, psi_rf) = (0.3, 0.4, 0.5)`.
> 
>     * Dalitz point
> 
> 
> ```julia
> σs = ((26.57159046, 17.86811729, 9.55283383))
> ```
> 
>     * spins projections, let's say `lam0=-1/2` `lam1,2,3 => 1/2, 0, 0`
> 
> 
> ## Amplitude
> 
> computed as `A_{c=3} + A_{c=1} * W_{1(3)}`
> 
> Values yielded by the formalisms printed for two terms separately:
> 
>     * DPD (gist https://gist.github.com/mmikhasenko/1d1f59664d1aefc3874cddab8670c736):
> 
> 
> ```
> term1 = -0.20903167421394864 - 0.9790486016833232im
> term2 = 3.704279900338359 - 2.486666937280139im
> ```
> 
>     * helicity
> 
> 
> ```
> term1: (xxx, xxx)
> term2: (xxx, xxx)
> ```
> 
>     * minus_phi
> 
> 
> ```
> term1: (xxx, xxx)
> term2: (xxx, xxx)
> ```
> 
>     * canonical
> 
> 
> ```
> term1: (xxx, xxx)
> term2: (xxx, xxx)
> ```

What do you understand as the two terms?
Is it A_{c=3} and A_{c=1} or A_{c=3} and A_{c=1} * W(1->3)?
Also what does A_{c=1} * W(1->3) mean? Is there an implicit sum over all final state helicities?

**Comment by KaiHabermann** [comment]
So for a single helicity amplitude I get:

```
Helicity
Term 1 (0.05627102784332641+0.014804674312629194j)
Term 2 0j
Minus Phi
Term 1 (0.020026947758740192-0.044971531972059886j)
Term 2 0j
```

**Comment by KaiHabermann** [comment]
Do you mean to write theta in the orientataion angles? Or is 0.4 really cos_theta?

**Comment by mmikhasenko** [comment]
I meant the cos

**Comment by mmikhasenko** [comment]
I have updated comment in the header to clarify the setup.

**Comment by mmikhasenko** [comment]
Closed in favor of #64
